skrf/media/media.py

```diff
@@ -31,11 +31,11 @@
 
 from ..constants import to_meters ,ZERO
 
-from abc import ABCMeta, abstractmethod, abstractproperty
+from abc import ABC, abstractmethod
 import re
 from copy import deepcopy as copy
 
-class Media(object):
+class Media(ABC):
```

Comment:
I wonder if some users have instantiated directly Media. After that, it will throw an error. @arsenovic, what do you think?

Reply:
Wasn't the Media class intended to be an abstract class?
There are abstract properties and the `__metaclass__`. 
I just wonder, why python let's me instantiate the previous code when Media is clearly an abstract class

Reply:
I guess it was indeed intended to be an abstract class (@arsenovic could say more than me.). I think it's more "pythonic" the way you propose, I was just wondering myself

Reply:
this change makes sense. i do believe the media is an abstract class, not sure why i didnt subclass abc. 
i think the closest thing to instantiate directly is DefinedGammaZ0.